docs/annexes/pkg-url-specification.md

```diff
@@ -258,7 +258,7 @@ To build a _purl_ string from its components:
     1. Append `?` to the _purl_
     1. Discard any pair where the value is empty
     1. Encode each value in UTF-8-encoding
-    1. If the key is `checksum` and there are more than one checksums, join the list with `,` to create the qualifier value
+    1. If the key is `checksum` and there are more than one checksum, join the list with `,` to create the qualifier value
```

Comment:
Since it's "more than one", the plural form is correct, in my opinion.

"There are more than one players in the field" sounds correct to me.